### PR TITLE
Feature/ Implemented Add property home screen

### PR DESCRIPTION
**Pull request by BhavaniMM**
Description:
Implemented Add property home screen

Testing:
IOS:
Screenshot 2023-11-07 at 6 42 21 PM

Android:
Screenshot 2023-11-07 at 2 24 22 PM

### CODE REVIEW

**Comment by Ganesh-softsuave** [comment]
@BhavaniMM  Please check why font family not applying

File: src/screens/AddProperty/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const themeContext: ThemeContextType = useThemeContext();
+  const styles: AddPropertySTyleSheet = style(themeContext.color);
+  const {t} = useTranslation();
+
+  return (
+    <>
+      {/* the below view to be removed after adding header */}
+      <View style={{height: 76}} />
+      <View style={styles.userInfo}>
+        <Image
```
Comment (Ganesh-softsuave) [inline src/screens/AddProperty/index.tsx:26]: Please create separate component for User Information

File: src/screens/AddProperty/index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          </CustomText>
+        </View>
+      </View>
+
+      <CustomText
+        style={styles.addPropertyText}
+        {...testID('add_property_screen_add_property_text')}>
+        {t('please_add_your_property')}
+      </CustomText>
+      <Image
```
Comment (Ganesh-softsuave) [inline src/screens/AddProperty/index.tsx:50]: Please use flex direction row for this Content. avoid using position absolute here

File: src/components/UserInfo/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {View, Image} from 'react-native';
+import {t} from 'i18next';
+import {ThemeContextType} from '@src/context/types';
+import {useThemeContext} from '@src/context/themeContext';
+import {testID} from '@src/utils/TestUtils';
+import profile from '@src/assets/images/Profile.png';
+import style from './styles';
+import {UserInfoStyleSheet} from './types';
+import CustomText from '../CustomTextField/CustomTextField';
```
Comment (Ganesh-softsuave) [inline src/components/UserInfo/index.tsx:10]: please import absolute path

File: src/screens/AddProperty/index.tsx [hunk 4]
```diff
@@ -19,39 +19,23 @@ function AddProperty(): JSX.Element {
   const {t} = useTranslation();
 
   return (
-    <>
+    <View style={styles.mainContainer}>
       {/* the below view to be removed after adding header */}
```
Comment (Ganesh-softsuave) [inline src/screens/AddProperty/index.tsx:23]: Please remove the commented line

File: src/screens/AddProperty/index.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        source={addPropertyImg}
+        style={styles.addPropertyImg}
+        {...testID('add_property_screen_add_property_image')}
+      />
+      <View style={styles.bottomView}>
+        <CustomButton
+          buttonStyle={styles.addPropertBtn}
+          labelText={t('add_your_first_property')}
+          accessibilityLabel={'add_property_add_first_property_button'}
+          onPress={() => {}}
```
Comment (Ganesh-softsuave) [inline src/screens/AddProperty/index.tsx:49]: Please create one function in viewmodel  comment over the function

**Review by Ganesh-softsuave — CHANGES_REQUESTED** [review]

**Comment by BhavaniMM** [comment]
> @BhavaniMM Please check why font family not applying

File: src/Navigator/MainNavigator/Constant.ts [hunk 6]
```diff
@@ -1,9 +1,11 @@
 import ScreenNameConstant from '@src/Navigator/MainNavigator/ScreenNameConstant';
 import Home from '@src/screens/Home';
+import AddProperty from '@src/screens/AddProperty';
 import {MainNavigatorConstantType} from './types';
 
 const MainNavigatorConstant: MainNavigatorConstantType = [
   {key: 1, name: ScreenNameConstant.HOME_SCREEN, Component: Home},
+  {key: 2, name: ScreenNameConstant.ADD_PROPERTY, Component: AddProperty},
```
Comment (Ganesh-softsuave) [inline src/Navigator/MainNavigator/Constant.ts:8]: Please create one more navigator in the name Owner Navigator as same as Auth Navigator move all  your  screens inside  that

File: src/Navigator/MainNavigator/ScreenNameConstant.ts [hunk 7]
```diff
@@ -2,6 +2,7 @@ import {ScreenNameType} from './types';
 
 const ScreenName: ScreenNameType = {
   HOME_SCREEN: 'Home',
+  ADD_PROPERTY: 'AddProperty',
```
Comment (Ganesh-softsuave) [inline src/Navigator/MainNavigator/ScreenNameConstant.ts:5]: move your screens to Owner navigator

**Review by Ganesh-softsuave — CHANGES_REQUESTED** [review]

File: src/screens/AddProperty/ViewModel.ts [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import {ViewModelReturnType} from './types';
+
+export default function ViewModel(): ViewModelReturnType {
```
[the remark this answers is not in the document]
Comment (BhavaniMM) [reply src/screens/AddProperty/ViewModel.ts:3]: It is mentioned as ViewModelReturnType

File: src/Navigator/OwnerNavigator/Constant.ts [hunk 9]
```diff
@@ -0,0 +1,9 @@
+import ScreenNameConstant from '@src/Navigator/OwnerNavigator/ScreenNameConstant';
+import AddProperty from '@src/screens/AddProperty';
+import {OwnerNavigatorConstantType} from './types';
+
+const MainNavigatorConstant: OwnerNavigatorConstantType = [
+  {key: 2, name: ScreenNameConstant.ADD_PROPERTY, Component: AddProperty},
```
Comment (Ganesh-softsuave) [inline src/Navigator/OwnerNavigator/Constant.ts:6]: Please change key as 1

**Review by Ganesh-softsuave — APPROVED** [review]

**Review by Ganesh-softsuave — APPROVED** [review]

**Pull request merged**